components/Administrator.vue

```diff
@@ -123,7 +136,9 @@ export default {
       this.validateForm();
     },
     validateForm() {
-      if(!this.$v.email.value.$invalid && !this.$v.password.value.$invalid && !this.$v.verify.value.$invalid) {
+      if(!this.$v.email.value.$invalid
```

Comment:
Can the logic here be inverted to something like.  I can't see the whole routine here, but there's too much negation going on here :)
```
if(a.$invalid || b$invalid || c.$invalid) {
  // do some error handling
  return; // return early.
}
// everything is OK, proceed
```

Reply:
Updated.